stingray/events.py

```diff
@@ -391,15 +394,17 @@ def read(filename, format_='pickle'):
             The :class:`EventList` object reconstructed from file
         """
 
+        if format_ == 'ascii':
+            from astropy.timeseries import TimeSeries
```

Comment:
why's this import here rather than at the top of the module?

Reply:
This one is important, because not all supported Astropy versions have TimeSeries